.gitignore

```diff
@@ -6,6 +6,7 @@ bin
 dist
 output
 strongswan
+vendor
```

Comment:
This causes `scripts/vendor` to be ignored too; you’ll need to specify `/vendor` instead. `/vendor/` would be even better, that will only match a directory.

Reply:
What is scripts/vendor for and why does "gitignoring" it break e2e? There is no scripts/vendor checked it.

Reply:
ohhhh! thanks, fixing.

Reply:
`make test` is broken too. I suspect `scripts/vendor` is what updates `vendor` locally; because it’s gitignored, it wasn’t included in Miguel’s commit, and therefore didn’t make it into this PR.

Reply:
oh I see - `scripts/vendor` is a new file.